bundle/edu.gemini.model.p1.submit/src/main/scala/edu/gemini/model/p1/submit/package.scala

```diff
@@ -19,8 +19,8 @@ package object submit {
     Map(LargeProgram -> url("large_program"), FastTurnaroundProgram -> url("fast_turnaround"))
   }
 
-  val productionSubmissionUrls: Map[SubmitDestination, String] = submissionUrls("http://hbfphase1-lv1.hi.gemini.edu/cgi-bin/backend/production")
+  val productionSubmissionUrls: Map[SubmitDestination, String] = submissionUrls("https://phase1.gemini.edu/cgi-bin/backend/production")
```

Comment:
I suggest to put the protocol/host name in a separate val, and then use that value to compose the full URL here and in the next line. That way if it is necessary to change it again, the change will be in only one place.